mindsdb/integrations/handlers/monkeylearn_handler/monkeylearn_handler.py

```diff
@@ -0,0 +1,79 @@
+from typing import Optional, Dict
+import pandas as pd
+import requests
+
+from monkeylearn import MonkeyLearn
+
+from mindsdb.integrations.libs.base import BaseMLEngine
+
+
+class monkeylearnHandler(BaseMLEngine):
+    name = "monkeylearn"
+
+    @staticmethod
+    def create_validations(self,args=None,**kwargs):
+
+        if "using" in args:
+            args = args["using"]
+
+        if "api_key" not in args:
+            raise Exception("API_KEY not found")
+        api_key = args["api_key"]
+        if "model_id" in args:
+            if "cl_" not in args["model_id"]:
+                raise Exception("Classifier tasks are only supported currently")
+        else:
+            raise Exception("Enter the model_id of model you want use")
+        model_id = args["model_id"]
+        # Check whether the model_id given by user exists in the user account or monkeylearn pre-trained models
+        url = 'https://api.monkeylearn.com/v3/classifiers/'
+        response = requests.get(url, headers={'Authorization': 'Token {}'.format(api_key)})
+        if response.status_code == 200:
+            models = response.json()
+            models_list = [model['id'] for model in models]
+        else:
+            raise Exception(f"Server response {response.status_code}")
+
+        if model_id not in models_list:
+            raise Exception(f"Model_id {args['model_id']} not found in MonkeyLearn pre-trained models")
+
+    def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[Dict] = None) -> None:
+        if "using" in args:
+            args = args['using']
+
+        self.model_storage.json_set('args', args)
+
+    def predict(self, df, args=None):
+        args = self.model_storage.json_get('args')
+        input_column = args['input_column']
+        if input_column not in df.columns:
+            raise RuntimeError(f"input columns {input_column} not found ")
+        input_list = df[input_column]
+        if len(input_list) > 500:
+            raise Exception("Classifier only supports 500 data elements in list")
```

Comment:
What's the reason behind this limitation? Does it have to do with the user's account plan from Monkeylearn? 

Please document this in the `README`.

Reply:
The Classifier API supports list of 500 data elements only. Updated the README accordingly.
<img width="590" alt="Screenshot 2023-05-29 at 11 11 43 PM" src="https://github.com/mindsdb/mindsdb/assets/94780581/8ca0a6dc-f241-4d38-8ca0-8ca9e23dea8a">